packages/common/src/lib/format-appeal-details.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @param {import("../client/appeals-api-client").AppealCaseWithAppellant} caseData
+ */
+exports.formatAgentDetails = (caseData) => {
+	const agentName = `${caseData.yourFirstName} ${caseData.yourLastName}`;
+
+	const agentDetails = caseData.yourCompanyName
+		? `${agentName}<br>${caseData.yourCompanyName}`
+		: agentName;
+
+	return agentDetails;
+};
+
+/**
+ * @param {import("../client/appeals-api-client").AppealCaseWithAppellant} caseData
+ */
+exports.formatVisibility = (caseData) => {
+	const visibility = caseData.appellantSiteAccess ? 'Yes' : 'No';
+
+	if (caseData.appellantSiteAccessDetails) {
+		return `${visibility}<br>${caseData.appellantSiteAccessDetails}`;
+	}
+
+	return visibility;
+};
```

Comment:
It's possible to do this kind of thing without branching but this is quite terse so feel free to ignore

```suggestion
exports.formatVisibility = (caseData) => {
	const visibility = caseData.appellantSiteAccess ? 'Yes' : 'No';

	return visibility + (caseData.appellantSiteAccessDetails && `<br>${caseData.appellantSiteAccessDetails}`);
};
```

<img width="217" alt="Screenshot 2024-03-08 at 14 17 29" src="https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/35689158/3d40ee3d-593a-4678-bee3-f4d74f35cb6c">

Reply:
Ah nice, have refactored to use that here, in the formatSafety and in the formatAgent functions